src/routes/auth.ts

```diff
@@ -53,10 +56,11 @@ auth.get('/callback', (req: Request, res: Response) => {
     }
 
     logger.debug('User successfully logged in');
-    res.redirect('/');
+    res.redirect(`/${req.language}`);
 });
 
-auth.get('/logout', (req: Request, res: Response) => {
-    res.clearCookie('jwt');
-    res.redirect('/auth/login');
+auth.get('/logout', (req: AuthedRequest, res: Response) => {
+    logger.debug('logging out user');
+    res.clearCookie('jwt', { domain: cookieDomain });
```

Comment:
the auth cookie previously wasn't clearing properly on logout - setting the domain seems to fix it.